MessengerOnFileForLinux/MessageFramework/test/MessageTest.cpp

```diff
@@ -1,11 +1,36 @@
 #include <MessageTestCore.hpp>
+#include <regex>
+#include <cstring>
+#include <iostream>
 
-TEST_F(MessageFixture, isMessageCorrectCreatedFromFullMessageInRow)
+TEST_F(MessageFixture, isMessageCorrectCreatedFromParameters)
 {
-    EXPECT_TRUE(true);
+         const std::regex regexReadMessage("\\[0]\\[20([0-9]{2})-([0-9]{2})-([0-9]{2}) ([0-9]{2}):([0-9]{2}):([0-9]{2})]\\[mnurzyns]\\[test read message]");
```

Comment:
formatowanie

Reply:
specjalnie zrobione, bo jest czytelniejsze -> możesz w łatwy sposób porównać sobie 3 regexy, że są prawie takie same.

Reply:
chodzi mi o taby przed constem

Reply:
mam ci napisać jeszcze raz to samo? Te consty są wyrównane do średników, a nie do lewej linii - powód masz powyżej.